src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/NotifyAppointmentServiceTest.kt

```diff
@@ -37,6 +37,7 @@ class NotifyAppointmentServiceTest {
 
   private fun notifyService(): NotifyAppointmentService {
     return NotifyAppointmentService(
+      "template",
```

Comment:
I would have a different value for `template` here to confirm we're using the right templateId on line 116

Reply:
done